dev-java/jakarta-activation-api/jakarta-activation-api-2.0.1-r1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/eclipse-ee4j/jaf/archive/refs/tags/2.0.1.tar.gz --slot 0 --keywords "~amd64 ~arm ~arm64 ~ppc64 ~x86" --ebuild jakarta-activation-2.0.1.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="jakarta.activation:jakarta.activation-api:2.0.1"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jakarta Activation API jar"
+HOMEPAGE="https://eclipse-ee4j.github.io/jaf/"
+SRC_URI="https://github.com/eclipse-ee4j/jaf/archive/refs/tags/${PV}.tar.gz -> jakarta-activation-${PV}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+DEPEND="virtual/jdk:1.8"
+RDEPEND="virtual/jre:1.8"
```

Comment:
these two lines will cause `module-info.java` is never compiled, even for those that have jdk 11 enabled.

Reply:
misunderstanding, sorry.
Re-done.